src/App.js

```diff
@@ -21,7 +21,7 @@ export default function App() {
       operation.setContext({
         headers: {
           Authorization: `Bearer ${authObject.accessToken}`,
-          "X-Boxwise": authObject.idTokenPayload.email,
+          "X-Boxwise": authObject.idToken,
```

Comment:
I'm a bit confused - isn't this header now redundant, if the claim is in the access token?

Reply:
Having checked the latest files in https://github.com/boxwise/boxwise-flask/pull/16/, I'm reasonably sure this is now redundant? :)

Reply:
yep, good catch :)